app/js/controllers/edit-ctrl.js

```diff
@@ -22,10 +22,15 @@ angular
 
 	$scope.updateResource = function(model, rdesc) {
 		var finalModel = relink(angular.copy(Restangular.stripRestangular(model)), rdesc);
-		delete finalModel.modified;
-		delete finalModel.created;
-
-		console.log(finalModel);
+		delete finalModel.attributes.modified;
+		delete finalModel.attributes.created;
```

Comment:
I'm pretty sure we don't need these two `delete` lines. 

The API is smart enough to ignore any provided modified date anyway, even though that isn't yet marked as readOnly, so it's ok to PATCH that back for now. And once `modified` is marked as readOnly, it'll automatically be excluded from the PATCH by the lines below.

Likewise with create: PATCHing back the same value won't be a problem for now, and when we mark it correctly as `readOnly` in the API, it'll automatically get handled here.

So, given that these two fields aren't causing any problems, we might as well keep your code cleaner by not having the `delete` statements and the special-casing they represent.